pandas/tests/plotting/test_converter.py

```diff
@@ -28,6 +28,7 @@
 pytest.importorskip("matplotlib.pyplot")
 
 
+@pytest.mark.skip(reason="There is no warnings in some versions.")
```

Comment:
We'll need to remove this skip. Any idea why it was failing?

Do you see the warning locally when you're on this branch?

Reply:
@TomAugspurger I tried to understand why it failed, but could find a correlation with what I changed in `test_boxplot_method.py`, In the meantime, I could not see any warning executing this function, with `test -vv` it shows there is no warning, so returns empty. And I open a notebook and run the code on this branch, i also could not see any warning. Do you have any idea why this should raise a warning? this code looks okay to me

![Screen Shot 2019-08-30 at 7 53 33 PM](https://user-images.githubusercontent.com/9269816/64041496-e375f180-cb5f-11e9-94ee-da7c0e9d207f.png)

Reply:
I am not sure if it is because of `backend` setup change?

Reply:
I'm also not sure why it would have changed.

> I am not sure if it is because of backend setup change?

Do you mean a different backend in the notebook? I wouldn't expect that to matter.

Reply:
this is where the warning defined: https://github.com/pandas-dev/pandas/blob/e55b6980fd5c99087d0959952b22e74434e9f5e2/pandas/plotting/_matplotlib/converter.py#L89
i remembered there was a PR about this merged before 0.25.1 was released, lemme check it out @TomAugspurger

Reply:
seems this was mentioned in this PR, but very likely not relevant though. @TomAugspurger 

https://github.com/pandas-dev/pandas/pull/27481

Reply:
Can you remove it and see if CI fails?

Reply:
yes, removed and seems still this test failed @TomAugspurger

Reply:
yes, removed and seems still this test failed @TomAugspurger